docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -114,14 +114,16 @@ Multiple `NuGet.Config` files allow you to store settings in different locations
 These settings collectively apply to any NuGet operation invoked from the command line or from Visual Studio, with settings that exist "closest" to a solution or the current folder taking precedence.
 If a command line tool is used on a project file, rather than a solution file, then the project directory is used as the "solution directory", which can lead to inconsistent behaviour when there is a `NuGet.Config` file in a subdirectory of the solution file.
 
-Specifically, NuGet loads settings from the different config files in the following order:
+Specifically, when `-configFile` command line option is not specified, NuGet loads settings from the different config files in the following order:
```

Comment:
The config file option differs across different tools, for example `-configFile` for nuget.exe, `--configfile` for dotnet.exe and `RestoreConfigFile` for msbuild. 

Maybe keep the sentence more generic rather than call out a specific option?

Reply:
Hm. I wasn't even aware that `dotnet` had a `--configfile` option.

Going to remove this change, and be more explicit about this being a `nuget.exe` specific behaviour below.

Reply:
All CLI tools have the options. 

How about saying `when a config file is not specified explicitly on the commandline`

That way it's more generic, since these docs apply to all tools.

Reply:
Had a deeper look through the docs to find where `--configfile` is for `dotnet`.

I do agree a more generic text would be more appropriate.

How about this latest change?